### ISSUE TITLE
throw new TypeError(`Unknown node of type: ${node.type}`);

### ISSUE THREAD

**Issue by mhaagens** [FormidableLabs/rapscallion#35]
Probably me doing something wrong, but I'm getting this error now;
```
/Users/user/project/node_modules/rapscallion/src/render/traverse.js:212

throw new TypeError(`Unknown node of type: ${node.type}`);TypeError: Unknown node of type: undefined
```

**Comment by aweary** [comment]
@mhaagens thanks for the report! Can you share the component you're trying to render when you get that error, or preferably a simplified example if it's part of a large render tree?

**Comment by divmain** [comment]
@mhaagens, if you're able to come up with a minimal reproduction, can you update the [repo](https://github.com/divmain/rapscallion-rr4) we were working from earlier?  You should still have push rights.

**Comment by mhaagens** [comment]
Sure, not able to right now but I can get on it tomorrow.

**Comment by divmain** [comment]
@mhaagens, this may have been related to children elements that are arrays.  I've pushed a fix and published `v2.0.1`.  Let me know if that resolves the issue for you, and I'll close this out.

```javascript
const Foo = () => (
  <div>
    <h1 id="foobar">Hello, world</h1>
  </div>
);

const Bar = () => (
  <div>
    <div id="sibling" />
    {
      range(5).map(idx =>
        <Foo key={idx} />
      )
    }
  </div>
);
```

**Comment by mhaagens** [comment]
Yep, that fixed it, thanks!
